src/components/cc-product-list/cc-product-list.js

```diff
@@ -0,0 +1,232 @@
+          ${categories.map((c) => this._renderCategory(c.categoryName, c.categoryName, c.toggled))}
+        </fieldset>
+      </div>
+      <div class="products">
+        ${this._renderProductsList()}
+      </div>
+    `;
+  }
+
+  /**
+   * @param {string} label
+   * @param {string} value
+   * @param {boolean} isToggled
+   * @returns {TemplateResult}
+   */
+  _renderCategory (label, value, isToggled) {
+    const id = label.replace(' ', '-');
+    return html`
+      <input 
+        type="radio"
+        id="${id}"
+        .value=${value}
+        name="category-filter"
+        class="visually-hidden" 
+        @change=${this._onCategoryChange}>
+      <label for="${id}">
+        <cc-badge
+          weight="${(isToggled ? 'dimmed' : 'outlined')}"
+          intent="info"
+        >${label}
+        </cc-badge>
+      </label>`;
+  }
+
+  _renderProductsList () {
+
+    const categoryDataList = this._productsCrtl.getCategoryDataList();
+
+    if (categoryDataList.length === 0) {
+      return html`
+      <p class="search-empty">${i18n('cc-product-list.search-empty')}</p>
+    `;
+    }
+
+    return categoryDataList.map((categoryData) => html`
+      <div class="category">
+        <div class="category-title">
+          ${categoryData.icon != null ? html`
+            <cc-icon .icon="${categoryData.icon}" class="category-icon"></cc-icon>
+          ` : ''}
+          <div class="category-name">${categoryData.categoryName}</div>
+        </div>
+        <div class="category-products">
+          ${categoryData.products.map((p) => html`
+            <cc-product-card
+              name="${p.name}"
+              description="${p.description}"
+              .keywords="${p.keywords ?? []}"
+              icon-url="${p.iconUrl}"
+              url="${p.url}"
+            ></cc-product-card>
+          `)}
+        </div>
+      </div>
+    `);
+  }
+
+  static get styles () {
+    return [
+      accessibilityStyles,
+      linkStyles,
+      // language=CSS
+      css`
+        :host {
+          display: block;
+        }
+
+        .category-name {
+          font-size: 1.75em;
+          font-weight: bold;
+        }
+
+        .category-products {
+          display: grid;
+          gap: 1em;
+          grid-template-columns: repeat(auto-fill, minmax(20em, 1fr));
+        }
+
+        .category-title {
+          display: flex;
+          align-items: center;
+          margin: 2.5em 0 1.5em;
+          gap: 0.5em;
+        }
+
+        .category-filter {
+          display: flex;
+          flex-wrap: wrap;
+          padding: 0;
+          border: none;
+          gap: 0.25em;
+        }
+
+        label:hover {
+          cursor: pointer;
+
+          --cc-color-bg-primary: var(--cc-color-text-primary-strong);
+        }
+
+        input[type='radio']:focus-visible + label {
+          /* TODO: change this if we add a custom prop to the cc-badge */
```

Comment:
**suggestion:** 
```suggestion
          /* 
           * FIXME:
           * remove this once the border-radius of the `cc-badge` component is set on its host instead of its wrapper
           * see https://github.com/CleverCloud/clever-components/issues/990.
           */
```